app/bot/llm.py

```diff
@@ -28,10 +28,13 @@ async def generate_response(
             try:
                 async with cs.post(url, json={
                     "model": _botconf.botconfig.llm_model,
-                    "prompt": message.content,
-                    "system": system_prompt,
-                    "context": [],
                     "stream": False,
+                    "messages": [
+                        {"role": "system",
+                        "content": system_prompt},
+                        {"role": "user",
+                        "content": message.content},
+                    ],
```

Comment:
Trailing comma, but I suspect Python is like most languages and doesn't care

Reply:
Nope, it doesn't. I also think they look better

Reply:
Fair enough, lol